test/h5test.c

```diff
@@ -772,6 +779,241 @@ h5_fileaccess_flags(unsigned flags)
     return H5I_INVALID_HID;
 } /* end h5_fileaccess_flags() */
 
+/*-------------------------------------------------------------------------
+ * Function:    h5_get_vfd_fapl
+ *
+ * Purpose:     Sets the file driver for a FAPL according to the value
+ *              specified in the constant "HDF5_DRIVER" or environment
```

Comment:
Is this still supposed to listen to the HDF5_DRIVER macro?

Reply:
No I missed updating that comment